torch_frame/gbdt/tuned_xgboost.py

```diff
@@ -93,6 +93,7 @@ def objective(
         dtrain: Any,  # xgboost.DMatrix
         dvalid: Any,  # xgboost.DMatrix
         num_boost_round: int,
+        early_stopping_rounds: int,
```

Comment:
nit: shall we set the default value to `50` for backward compatibility?

Reply:
Yes. I set the `early_stopping_rounds` to default to 50